README.md

```diff
@@ -1 +1,24 @@
-# notes
\ No newline at end of file
+# LAB - Class 01
+
+Project: Notesy
+Author: Ben Hill
+Links and Resources
+submission PR
+ci/cd (GitHub Actions)
+back-end server url (when applicable)
+front-end application (when applicable)
```

Comment:
These should be links for example `[submission RR](https://github.com/benhill-401-advanced-javascript/notes/pull/1)` which will turn into this [submission RR](https://github.com/benhill-401-advanced-javascript/notes/pull/1). That way you can just turn in the README click on the links to see your PR, test, and later deployments. 

Also in the README add a space between each new line.

Reply:
Added!